### ISSUE TITLE
Access blocked: Polybar Gmail’s request is invalid

### ISSUE THREAD

**Issue by Ulrar** [vyachkonovalov/polybar-gmail#23]
Hi,

Google just forced me to change my password for some reason so the credentials.json file I had is now invalid.
Trying to re-generate it, the request link it gives me returns this : 

```
Access blocked: Polybar Gmail’s request is invalid
my email
You can’t sign in because Polybar Gmail sent an invalid request. You can try again later, or contact the [developer](https://accounts.google.com/) about this issue. [Learn more about this error](https://support.google.com/accounts/answer/12379384)
If you are a developer of Polybar Gmail, see [error details](https://accounts.google.com/).
Error 400: invalid_request
```

Clicking on that link gives this : 
```
The out-of-band (OOB) flow has been blocked in order to keep users secure. Follow the Out-of-Band (OOB) flow migration guide linked in the developer docs below to migrate your app to an alternative method.
Request details: redirect_uri=urn:ietf:wg:oauth:2.0:oob 
```

I tried updating some pip packages in case it might be an old library issue but no changes, not sure what I may be missing.
Or has there been an actual change to this on Google's side recently ?

**Comment by crabvk** [comment]
Yup, Google has finally deprecated authorization method which polybar-gmail uses (in October, I believe).
I'm gonna try to fix it.

**Comment by crabvk** [comment]
Fixed in e47dcc6a45cffe0fbaf0ae6b7597631353bb7599, please check auth again.

**Comment by Ulrar** [comment]
That looks good, thanks for the quick fix !
